chroma_core/models/stratagem.py

```diff
@@ -87,10 +112,10 @@ def run(self, kwargs):
                 "Description=Start Stratagem run on {}\n"
                 "\n[Timer]\n"
                 "OnBootSec={}\n"
-                "OnUnitActiveSec={}\n".format(config.filesystem_id, config.interval / 1000, config.interval / 1000)
+                "OnUnitActiveSec={}\n".format(config.filesystem.id, config.interval / 1000, config.interval / 1000)
```

Comment:
Can we switch config.filesystem.id to config.filesystem.name?  I know this preserves functionality, but it'll be nicer for end-user